README.md

```diff
@@ -8,32 +8,68 @@ Learn more about the RFC proposal process [here](https://github.com/polkadot-fel
 
 ## Usage
 
+On an RFC Pull Request, add a comment starting with `/rfc`.
+
+### Commands
+
+1. Propose
+
+Proposes the creation of a referendum aiming to approve the given RFC.
+
+```
+/rfc propose
+```
+
+Will result in a comment response with instructions to create an on-chain referendum.
+
+2. Process
+
+After the RFC referendum was confirmed, it processes the Pull Request (by merging or closing it).
+
+```
+/rfc process <block hash of when the referendum was confirmed>
+```
+
+If you're not sure where to get this block hash,
+send a `/rfc process` command and the action will respond with more instructions.
```

Comment:
how do people learn about existing commands within repo and its interface? 

does it make sense to output this when someone types `/rfc` or `/rfc help` or they can head here to README from somewhere else?

Reply:
Added some instructions: [preview 1](https://github.com/paritytech-stg/RFCs/pull/9#issuecomment-1710003130), [preview 2](https://github.com/paritytech-stg/RFCs/pull/9#issuecomment-1710003752).